### DISCUSSION TITLE
[el-pagination] total - Format the "total" number with thousand separator

### DISCUSSION THREAD

**Discussion by markopetrovic** [element-plus/element-plus#10293] [Feature Request]
It would be nice if we could pass a function to display the "total" number in the desired format or separate it with a thousand separator by default.

 ![el-pagination-with-thousands-separator](https://user-images.githubusercontent.com/5808218/198380372-9bbf5621-295d-4c14-ba34-bde367e90863.png)

**Comment by baiwusanyu-c** [comment]
I agree with your idea. Because as far as I know, different from mainland China, many countries have different usage habits for numbers, including the use of thousand separators. 
I created a PR see [#10322 ](https://github.com/element-plus/element-plus/pull/10322)

**Reply by markopetrovic** [reply]
Thank you very much for the PR and your quick response!
Hopefully, the element team will agree and accept the changes.
